projects/update_ping_ready_nightly_validation.kp/knowledge.md

```diff
@@ -0,0 +1,347 @@
+    Target buildId mismatch: 20170804100354 must be more recent than 20170805100334:	0.002%
+    Target buildId mismatch: 20170804100354 must be more recent than 20170806100257:	0.002%
+    Target buildId mismatch: 20170804100354 must be more recent than 20170807113452:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170805100334:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170806100257:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170807113452:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170808100224:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170809100326:	0.001%
+    Target buildId mismatch: 20170804193726 must be more recent than 20170810100255:	0.001%
+    Target buildId mismatch: 20170805100334 must be more recent than 20170806100257:	0.001%
+    Target buildId mismatch: 20170805100334 must be more recent than 20170807113452:	0.001%
+    Target buildId mismatch: 20170805100334 must be more recent than 20170809100326:	0.001%
+    Target buildId mismatch: 20170806100257 must be more recent than 20170807113452:	0.002%
+    Target buildId mismatch: 20170806100257 must be more recent than 20170808114032:	0.001%
+    Target buildId mismatch: 20170806100257 must be more recent than 20170810100255:	0.001%
+    Target buildId mismatch: 20170806100257 must be more recent than 20170812100345:	0.001%
+    Target buildId mismatch: 20170807113452 must be more recent than 20170808114032:	0.002%
+    Target buildId mismatch: 20170807113452 must be more recent than 20170809100326:	0.003%
+    Target buildId mismatch: 20170807113452 must be more recent than 20170810100255:	0.002%
+    Target buildId mismatch: 20170808100224 must be more recent than 20170808114032:	0.001%
+    Target buildId mismatch: 20170808114032 must be more recent than 20170809100326:	0.005%
+    Target buildId mismatch: 20170808114032 must be more recent than 20170810100255:	0.001%
+    Target buildId mismatch: 20170809100326 must be more recent than 20170810100255:	0.002%
+    Target buildId mismatch: 20170809100326 must be more recent than 20170811100330:	0.001%
+    Target buildId mismatch: 20170810100255 must be more recent than 20170811100330:	0.001%
+    Target buildId mismatch: 20170810100255 must be more recent than 20170812100345:	0.001%
+    Target channel mismatch: expected nightly-cck-mint got nightly:	0.005%
+    Target channel mismatch: expected nightly-cck-rambler got nightly:	0.002%
+
+
+The vast majority of the data in the payload seems reasonable (99.71%). However, a handful of `update` pings are reporting a `targetBuildId` which is older than the current build reported by the ping's environment. Some pings are also reporting a `targetChannel` different than the one in the environment. These issues do not represent a problem, as most of the data is correct.
```

Comment:
"These issues do not represent a problem" Why? They seem like a problem to me. 

They shouldn't affect analysis as there are so few of them? And are probably not errors in payload because they could represent people specifically downgrading? (I don't fully understand the update ping mechanism. Didn't think downgrades were a thing that would show up)